geomesa-tools/README.md

```diff
@@ -498,55 +476,56 @@ The file type is inferred from the extension of the file, so ensure that the for
            Zookeepers (host[:port], comma separated)
 
 
+
 #### Example commands:
 
 ##### Ingest CSV with single WKT (Well Known Text) geometry
 
-    # file.csv (comma separated)
-    featureId,date,geom
-    1.23623623,01/01/2014 06:30:23,POINT(2 3)
-    290.43234,01/02/2014 08:35:24,POINT(3 3)
-    3.14159,01/03/2014 18:11:56,POINT(4 5)
+    $ cat example1.csv
+    ID,Name,Age,LastSeen,Friends,Lat,Lon
+    23623,Harry,20,2015-05-06,"Will, Mark, Suzan",-100.236523,23
+    26236,Hermoine,25,2015-06-07,"Edward, Bill, Harry",40.232,-53.2356
```

Comment:
It's *Hermione*, not Hermoine.

![leviosa-gif](https://i.imgur.com/qJQZko4.gif)